sympy/concrete/tests/test_sums_products.py

```diff
@@ -855,3 +855,39 @@ def test_indexed_idx_sum():
     raises(ValueError, lambda: Product(A[k], (k, 1, 4)))
     raises(ValueError, lambda: Product(A[k], (k, 0, 3)))
     raises(ValueError, lambda: Product(A[k], (k, 2, oo)))
+
+
+def test_is_convergent():
+    # divergence tests --
+    assert Sum(n/(2*n + 1), (n, 1, oo)).is_convergent() == S.false
+    assert Sum(factorial(n)/5**n, (n, 1, oo)).is_convergent() == S.false
+    assert Sum(3**(-2*n - 1)*n**n, (n, 1, oo)).is_convergent() == S.false
+
+    # root test --
+    assert Sum((-12)**n/n, (n, 1, oo)).is_convergent() == S.false
+    assert Sum(2**n/factorial(n), (n, 1, oo)).is_convergent() == S.true
+
+    # integral test --
+    assert Sum(1/(n*log(n)), (n, 2, oo)).is_convergent() == S.false
+
+    # comparison test --
+    assert Sum(1/n**(S(6)/5), (n, 1, oo)).is_convergent() == S.true
+    assert Sum(2/(n*sqrt(n - 1)), (n, 2, oo)).is_convergent() == S.true
+    assert Sum(1/(n**2 + 1), (n, 1, oo)).is_convergent() == S.true
+
+    # singularities
+    assert Sum(1/(n**3 - 1), (n, 0, oo)).is_convergent() == S.false
+
+    # alternating series tests --
+    assert Sum((-1)**(n - 1)/(n**2 - 1), (n, 3, oo)).is_convergent() == S.true
+    assert Sum((-1)**n*n, (n, 3, oo)).is_convergent() == S.false
+
+
+@XFAIL
+def test_convergent_failing():
+    assert Sum(1/(n + log(n)), (n, 1, oo)).is_convergent() == S.false
+
+    # currently raise TypeError
+    assert Sum(sin(n)/n**3, (n, 1, oo)).is_convergent() == S.true
+    assert Sum(ln(n)/n**3, (n, 1, oo)).is_convergent() == S.true
+    assert Sum(1/(n**2*log(n)), (n, 2, oo)).is_convergent() == S.true
```

Comment:
This works now, since of initial `Order` not able to handle i had added this. I will add to normal tests.